guides/common/modules/con_permissions-for-remote-execution.adoc

```diff
@@ -15,6 +15,13 @@ You can use the `view_hosts` and `view_smart_proxies` permissions to limit which
 The `execute_template_invocation` permission is a special permission that is checked immediately before execution of a job begins.
 This permission defines which job template you can run on a particular host.
 This allows for even more granularity when specifying permissions.
+
+ifdef::satellite[]
+The `execute_jobs_on_infrastructure_hosts` permission allows you to run remote execution jobs against {ProjectName} and {Capsules} registered as hosts to {ProjectName}.
```

Comment:
We don't use "allows you" in our docs. See the Anthropomorphism section in the IBM Style Guide. 
{Capsules} is not a real attribute. 
This could never have built correctly meaning you did not test your work before pushing. This is not the process that we have agreed on to contribute to this repository.

Reply:
I assumed since this is still in draft state that I would have some time to fix some issues before it was reviewed.
I'm aware of the build issues (I did build, but with errors I wanted to look at in the draft before getting a review), and I'm also aware of the items you've mentioned.

Reply:
Generally the process would be to not to push with build issues.
I am glad that you are aware of all the issues. 
The draft state in PR here in github would mainly be used for public collaboration of some kind. 
Looks like you have file conflicts and need to rebase.

Reply:
Thanks, @melcorr .  
I misunderstood the point of a draft PR, and I'll make sure to have things in better shape before pushing them at all in the future.
My rebase appears to have failed (as I did do that before trying to push anything, as I've learned from a number of other PR fails), but something clearly went wrong, and I'll be looking into that on Monday.
Thanks again for your notes.